controllers/admin/AdminProductsController.php

```diff
@@ -444,7 +444,7 @@ protected function loadObject($opt = false)
     {
         $result = parent::loadObject($opt);
         if ($result && Validate::isLoadedObject($this->object)) {
-            if (Shop::getContext() == Shop::CONTEXT_SHOP && Shop::isFeatureActive() && !$this->object->isAssociatedToShop()) {
+            if (Shop::isFeatureActive() && !$this->object->isAssociatedToShop()) {
```

Comment:
Hello,
I have know relaced the code and it works much better.

I have another problem: when I modify in shop3 a product which is active in shop 3 and not in the other shops, the product becomes activ in the other shops.